hack/install_metallb.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Why we need this file?

Reply:
I use this to test the new controller in standalone mode w/o using deployment CRD I could revert it now if u like

Reply:
Yes I don't feel it belongs here

Reply:
ok will remove it